src/entities/supportRequest.ts

```diff
@@ -18,6 +18,18 @@ export enum SupportRequestErrors {
   ExistingActiveRequest = 'ExistingActiveRequest',
 }
 
+const genHash = (): string => {
+  const prefix = Math.random()
+    .toString(36)
+    .substring(2, 15);
+
+  const suffix = Math.random()
+    .toString(36)
+    .substring(2, 15);
+
+  return prefix + suffix;
+};
```

Comment:
Could we refactor this into a util file? Looks like `team.ts` uses the same function